tst/regression/test_suites/restart/restart.py

```diff
@@ -53,7 +53,7 @@ def Analyse(self, parameters):
 
         varName = "/advected"
         goldData = gold[varName][:].flatten()
-        silverData = gold[varName][:].flatten()
+        silverData = silver[varName][:].flatten()
```

Comment:
This is a good and important fix. But it may conflict with the same fix in #330

Reply:
#330 makes a few more deeper changes to the structure and comparisons, and will override this one.  Without this fix in place I couldn't test which is why I made it here as well.